dotcom-rendering/cdk/lib/dotcom-rendering.ts

```diff
@@ -185,28 +192,23 @@ export class DotcomRendering extends GuStack {
 			reason: 'Retaining logical ID of resource created via CDK which cannot be changed easily',
 		});
 
+		const loggingStreamName = new GuStringParameter(
+			this,
+			'KinesisLoggingStreamName',
+			{ default: '/account/services/logging.stream.name', fromSSM: true },
+		).valueAsString;
```

Comment:
minor: There is a [construct in GuCDK](https://github.com/guardian/cdk/blob/main/src/constructs/core/parameters/log-shipping.ts) for this:

```ts
const loggingStreamName = GuLoggingStreamNameParameter.getInstance(this);
```